magic/loader.py

```diff
@@ -1,60 +1,85 @@
 from ctypes.util import find_library
 import ctypes
+import sys
 import glob
 import os
+import logging
 import subprocess
-import sys
+
+logger = logging.getLogger(__name__)
+here = os.path.dirname(__file__)
+
+def _lib_candidates_linux():
+    """Yield possible libmagic library names on Linux."""
+    prefixes = ('libmagic.so.1', 'libmagic.so')
+
+    for i in prefixes:
```

Comment:
This is not an integer (i).
```suggestion
    for prefix in prefixes:
```

Reply:
f7341ce